step-templates/ssis-deploy-ispac-from-referenced-package.json

```diff
@@ -3,15 +3,15 @@
     "Name": "Deploy ispac SSIS project from Referenced Package",
     "Description": "This step template will deploy SSIS ispac projects to SQL Server Integration Services Catalog.  The template uses a referenced package and is Worker compatible.\n\nThis template will install the Nuget package provider if it is not present on the machine it is running on.",
     "ActionType": "Octopus.Script",
-    "Version": 2,
+    "Version": 3,
     "CommunityActionTemplateId": null,
     "Packages": [
       {
         "Id": "7a1f5eb6-0b6a-4319-a0e8-7b4d13ea609e",
         "Name": "SSIS Package",
         "PackageId": "#{ssisPackageId}",
-        "FeedId": "feeds-builtin",
-        "AcquisitionLocation": "",
+        "FeedId": "#{TemplatePackageFeedId}feeds-builtin",
```

Comment:
Looks like you forgot to remove the previous value when inserting the variable, `feeds-builtin` shouldn't be there

Reply:
Good catch!  Removed :)